libcodechecker/util.py

```diff
@@ -662,3 +662,16 @@ def generate_session_token():
     Returns a random session token.
     """
     return uuid.UUID(bytes=os.urandom(16)).__str__().replace('-', '')
+
+
+def slugify_file_name(file_name):
```

Comment:
I think we could rename it to `slugify` only, the input can be run name, file path ...
Please update the function documentation according to this, `removes and replaces special characters in a given text`